frontend/src/util/webAudio.js

```diff
@@ -107,6 +107,11 @@ export const stopBuffer = () => {
 
 // play buffer by section.id
 export const playBuffer = (id) => {
+    // temporary solution for ios17
+    // https://devcodef1.com/news/1068842/fixing-ios-17-web-audio-issue
+    audioContext.close();
```

Comment:
Will this throw an error if `audioContext` isn't (yet) defined, or is that impossible to happen?